### PR TITLE
Add new guide how to enable pom plugin -  Closes #1256

### LINKED ISSUE #1256: New guide: How to enable the PoM plugin
### Description
How to enable the PoM plugin to automatically identify and punish delegate misbehavior in the network.

- mention, if the node restarts, you need to re-enable the plugin

### Additional information
If you take a look on this page you see how to structure a jsonrpc request https://www.jsonrpc.org/specification

I don't know if you looked here: https://lisk.com/documentation/lisk-sdk/references/lisk-framework/report-misbehavior-plugin.html but what the docs call actions are methods in jsonrpc terms

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1256

### CODE REVIEW

File: modules/ROOT/pages/management/enabling-misbehavior-report.adoc [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    "reportMisbehavior": {
+		"encryptedPassphrase": "iterations=1000000&cipherText=643bfbf1b6f1dc0ce740dd9fc9f27a682e476dc5de4e6c023deded4d3efe2822346226541106b42638db5ba46e0ae0a338cb78fb40bce67fdec7abbca68e20624fa6b0d7&iv=8a9c461744b9e70a8ba65edd&salt=3fe00b03d10b7002841857c1f028196e&tag=c57a798ef65f5a7be617d8737828fd58&version=1",
+	},
+  }
+}
+----
+
+TIP: See more available configuration options at the xref:{url_plugin_report_misbehavior}[] reference.
+
+== Restart the node with enabled plugin
```
Comment (chris529) [inline modules/ROOT/pages/management/enabling-misbehavior-report.adoc]:
```suggestion
== Restart the node with the enabled plugin
```

**Review by chris529 — COMMENTED** [review]
Super! only 1 minor point 👍

**Review by mehmetegemen — APPROVED** [review]
Great work! 🎉

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Pull request merged**